src/run.py

```diff
@@ -653,9 +654,12 @@ def get_diffs(self, pr: GitPullRequest, pr_url: str) -> list[FileDiff]:
 			return result
 
 		# Get the files changed.
-		pr_branch = branch_pat.sub('', pr.source_ref_name) # type: ignore
-		pr_branch = urllib.parse.quote(pr_branch)
-		target = GitTargetVersionDescriptor(target_version=pr_branch, target_version_type='branch')
+		if pr.status == 'completed':
+			target = GitTargetVersionDescriptor(target_version=latest_commit.commit_id, target_version_type='commit')
```

Comment:
I'm surprised this worked, I would have suspected that the `base` would need to change since the commit would be in the target branch for the PR.

Reply:
I think "target" in this context doesn't mean the target branch of the PR, but the PR branch (source branch).  The "base" here is main/master.

Reply:
Oh, wait, I see what you mean.  Yeah, it's strange that "main" or "master" as a branch name works for the "base".  Maybe it's because of `diff_common_commit=True`?  I imagine that does the same as using three dots instead of two dots between the branch names when using `git diff`, i.e. it compares the second ref to the merge-base of both refs rather than to the first ref directly.

Reply:
Yeah target and base get flipped around here. I had to re-read it a few times today to understand this again.

What I meant was that I'm surprised a good diff is found once the PR has been merged into the target (e.g., "main"). If you tell me that you tested it with PRs that have been merged days ago, or at least not the latest PR to be merged, then I believe you. Would you mind confirming that it works with a PR that was merged many days or even weeks ago and would have conflicts or where lines changed in the PR were changed by later PRs? I'm concerned that the diff will be wrong. You can also try printing the `diff_url` from below and seeing how it looks.

Reply:
I just tested it again and it works.  I'll send you the details privately because they're confidential.